src/commands/music/NowPlayingCommand.ts

```diff
@@ -0,0 +1,76 @@
+import { haveQueue } from "../../utils/decorators/MusicUtil";
+import { DefineCommand } from "../../utils/decorators/DefineCommand";
+import { CommandContext } from "../../structures/CommandContext";
+import { BaseCommand } from "../../structures/BaseCommand";
+import { IQueueSong } from "../../typings";
+import { createEmbed } from "../../utils/createEmbed";
+import { AudioPlayerPlayingState, AudioResource } from "@discordjs/voice";
+import { MessageActionRow, MessageButton, MessageEmbed } from "discord.js";
+
+@DefineCommand({
+    description: "Show the media player status",
+    name: "nowplaying",
+    slash: {
+        name: "nowplaying"
+    },
+    usage: "{prefix}nowplaying"
+})
+export class NowPlayingCommand extends BaseCommand {
+    @haveQueue()
+    public async execute(ctx: CommandContext): Promise<void> {
+        function getEmbed(): MessageEmbed {
```

Comment:
```suggestion
        private getEmbed(): MessageEmbed {
```

Reply:
Whut? Changing `function` to `private`? `private` is used if we're declaring a class method. `getEmbed` is just a normal function.